Core/clim-core/frames.lisp

```diff
@@ -1684,9 +1687,10 @@ have a `pointer-documentation-pane' as pointer documentation,
 	      (do-hilite last-presentation (last-window) :unhighlight))
 	    (setq last-presentation nil)
 	    (do-feedback window x y :highlight)
-	    (document-drag-n-drop translator nil
-				  context-type frame event window
-				  x y))
+	    (when translator
+              (document-drag-n-drop translator nil
```

Comment:
Is this possible that translator is NIL?

```
;; Try to run the feedback and highlight functions of the translator
	 ;; that got us here.
	 (translator (or (find translator-name translators :key #'name)
			 (car translators)))
```

Reply:
Likely. With the your suggested change to `document-drag-n-drop` it will be safe either way.